### PR TITLE
Dynamic data in templates

### PR DESCRIPTION
**Pull request by igorkruglyak**
Modified views to display posts from database

### CODE REVIEW

File: mysite/settings.py [hunk 1]
```diff
@@ -78,20 +78,28 @@
 
 AUTH_PASSWORD_VALIDATORS = [
     {
-        'NAME': 'django.contrib.auth.password_validation.\
-            UserAttributeSimilarityValidator',
+        'NAME': (
```
Comment (webknjaz) [inline mysite/settings.py]: Looks like you forgot to rebase on top of `origin/master` (did you update your local repo from github remote with `git fetch --all` or `git pull --rebase`?).

**Comment by webknjaz** [comment]
You need to fix the linter offenses and typos in var names. First of all, check that the code works by running your app because now it doesn't since you refer to a non-existing variable name.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -52,5 +52,6 @@ repos:
     files: \.py$
     entry: python3 -m pylint.__main__
     args:
+    - --load-plugins=pylint_django
```
Comment (webknjaz) [inline .pre-commit-config.yaml]: This should be submitted separately. It does not do "Dynamic data in templates"

**Comment by webknjaz** [comment]
@igorkruglyak

> Dynamic data in templates

What does this mean, again?

**Review by webknjaz — DISMISSED** [review]

File: blog/views.py [hunk 3]
```diff
@@ -1,7 +1,14 @@
 """Views for the blog app."""
 from django.shortcuts import render
+from django.utils import timezone
+from .models import Post
 
 
 def post_list(request):
     """Render post list on the webpage."""
-    return render(request, 'blog/post_list.html', {})
+    posts = (
```
Comment (webknjaz) [inline blog/views.py:9]:
```suggestion
    posts___ = (
```

**Review by webknjaz — APPROVED** [review]

File: blog/views.py [hunk 4]
```diff
@@ -1,7 +1,14 @@
 """Views for the blog app."""
 from django.shortcuts import render
+from django.utils import timezone
```
Comment (webknjaz) [inline blog/views.py:3]: comment1

File: blog/views.py [hunk 5]
```diff
@@ -1,7 +1,14 @@
...
 
 
 def post_list(request):
     """Render post list on the webpage."""
-    return render(request, 'blog/post_list.html', {})
+    posts = (
+        Post.objects.
+        filter(published_date__lte=timezone.now()).
+        order_by('published_date')
+    )
```
Comment (webknjaz) [inline blog/views.py:13]: comment2

**Review by webknjaz — CHANGES_REQUESTED** [review]
Well done!

**Review by webknjaz — APPROVED** [review]

**Review by slsh1o — APPROVED** [review]
Good job.

File: blog/views.py (hunk 5 above)
Comment (gaupt) [reply blog/views.py:13]: !!!

**Pull request merged**